src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -215,7 +216,6 @@ def _snql_on_tx_satisfaction_factory(
                                 ),
                             ],
                         ),
-                        Function("in", [Column("metric_id"), list(metric_ids)]),
```

Comment:
Changing the order of these parameters for an "and" operation makes no difference, however, it helps when we make this assertion later in this PR:
```
assert miserable_users(self.org_id, self.metric_ids, alias) == uniq_if_column_snql(...)
```